merkle-tree/config/default.js

```diff
@@ -3,33 +3,23 @@
 @author iAmMichaelConnor
```

Comment:
Ah, it looks like this whole config file has been overwritten with something nightfall-specific.
I added comments later in the file, but my overall recommendation would be to revert all changes to this file. This config should be a demonstration of all the config features of Timber, and information is getting lost with these changes.

Reply:
Ooops, yes.  I used a nightfall config to test it (because the existing tests don't exercise the db) then forgot to swap it back!  I'll push the original